components/vc/rear/src/HW/HW_gpio_componentSpecific.c

```diff
@@ -11,21 +11,196 @@
 #include "HW_gpio.h"
 
 const HW_GPIO_S HW_GPIO_pinmux[HW_GPIO_COUNT] = {
-    [HW_GPIO_CAN1_RX] = {
-        .port = GPIOB,
-        .pin = GPIO_PIN_8,
+    [HW_GPIO_DIG_SPARE1] = {
+        .port = GPIOE,
+        .pin = GPIO_PIN_2,
         .mode = GPIO_MODE_INPUT,
         .speed = GPIO_SPEED_FREQ_HIGH,
         .pull = GPIO_NOPULL,
         .resetState = HW_GPIO_NOSET,
     },
-    [HW_GPIO_CAN1_TX] = {
+    [HW_GPIO_DIG_SPARE2] = {
+        .port = GPIOE,
+        .pin = GPIO_PIN_3,
+        .mode = GPIO_MODE_INPUT,
+        .speed = GPIO_SPEED_FREQ_HIGH,
+        .pull = GPIO_NOPULL,
+        .resetState = HW_GPIO_NOSET,
+    },
+    [HW_GPIO_DIG_SPARE3] = {
+        .port = GPIOE,
+        .pin = GPIO_PIN_4,
+        .mode = GPIO_MODE_INPUT,
+        .speed = GPIO_SPEED_FREQ_HIGH,
+        .pull = GPIO_NOPULL,
+        .resetState = HW_GPIO_NOSET,
+    },
+    [HW_GPIO_DIG_SPARE4] = {
+        .port = GPIOE,
+        .pin = GPIO_PIN_5,
+        .mode = GPIO_MODE_INPUT,
+        .speed = GPIO_SPEED_FREQ_HIGH,
+        .resetState = HW_GPIO_NOSET,

```

Comment:
Missing pull. Will default to 0 which is not necessarily correct. Theres '-Wuninitialized' but id have to see the component impacts to turn it on or not